src/Home.vue

```diff
@@ -0,0 +1,103 @@
+<script setup lang="ts">
+import PageLink from './PageLink.vue';
+import CivImage from './CivImage.vue';
+import type { CivImageProps, PageLinkProps } from './objects';
+import NavBar from './components/ENNavBarES.vue';
+
+const links: PageLinkProps[] = [
+  {
+    text: 'English',
+    url: './english.html',
+  },
+  {
+    text: 'Español',
+    url: './spanish.html',
+  },
+];
+
+const images: CivImageProps[] = [
+  {
+    link: 'https://nmscd.com/',
+    imgAlt: 'NMSCD',
+    img: 'https://avatars.githubusercontent.com/u/62565964?s=280&v=4',
```

Comment:
higher resolution image
```suggestion
    img: 'https://raw.githubusercontent.com/NMSCD/About/master/logo/NMSCD.png',
```